From 84326e3a92d8dc7dfb1f5baad7ce01f35da0b51e Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Tue, 8 Oct 2019 16:51:31 +0800
Subject: [PATCH 2356/2367] isys & psys package lib2600b0 for commit-id 8a006fc

FW_Manifest_ID: 6f74becd9f5146e393b206b12e08b1ccdabac0de

FW_Build_Branch: stable_iot_gp1_ivi_20190410

FW_Build_Topic: auto_merge_queue

library version: commit-id 8a006fc

FW Bundle patch number of this release :  436061

FW tag : scci_stable_iot_gp1_ivi_20190410_20191003_1233

Change-Id: If3ac5df57983695bdfcd98e95ea2622267de672c
Depends-on: I3d0743ee17850afe440fff4ad381824a515e287e
Depends-on: I5eee1608099b2cea349575262ef5dd609ea7b18d
Depends-on: I247929b8a8a54fae94832784ae5b625344dac02b
Tracked-On: #PKT-2935
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
Signed-off-by: Ong Yang Yang <yang.yangx.ong@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index b2d4abc..a89c17f 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20190731
+#define IA_CSS_FW_PKG_RELEASE  0x20191003
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index b2d4abc..a89c17f 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20190731
+#define IA_CSS_FW_PKG_RELEASE  0x20191003
-- 
2.7.4

